raise Exception('None shall pass!')
